Formula/pinot.rb

```diff
@@ -0,0 +1,82 @@
+class Pinot < Formula
+  desc "Realtime distributed OLAP datastore"
+  homepage "https://pinot.apache.org/"
+  url "https://downloads.apache.org/pinot/apache-pinot-0.9.3/apache-pinot-0.9.3-bin.tar.gz"
+  sha256 "c253eb9ce93f11f368498229282846588f478cb6e0359e24167b13e97417c025"
+  license "Apache-2.0"
+  revision 1
+  head "https://github.com/apache/pinot.git", branch: "master"
+
+  depends_on "libtool" => :build
+
+  depends_on "openjdk"
+  depends_on "pinotbase"
+  depends_on "pinotbroker"
+  depends_on "pinotserver"
+  depends_on "zookeeper"
+
+  def default_pinot_env
+    <<~EOS
+      [ -z "$JAVA_OPTS" ] && export JAVA_OPTS="-Xms4G -Xmx8G -XX:+UseG1GC -XX:MaxGCPauseMillis=200 -Xloggc:gc-pinot-controller.log -Dlog4j2.configurationFile=/usr/local/etc/pinot/log4j2.xml"
+    EOS
+  end
+
+  def install
+    data = var/"lib"
+
+    inreplace "conf/pinot-controller.conf",
+      "controller.data.dir=/tmp/pinot/data/controller", "controller.data.dir=#{data}/pinot/data/controller"
+
+    inreplace "conf/pinot-controller.conf",
+      "pinot.zk.server", "controller.zk.str"
+
+    inreplace "conf/pinot-controller.conf",
+      "pinot.cluster.name", "controller.helix.cluster.name"
+
+    mv "conf", "pinot"
+    etc.install "pinot"
+    libexec.install_symlink etc/"pinot" => "conf"
+
+    (var+"lib/pinot/data/controller").mkpath
+  end
+
+  plist_options manual: "pinot-admin StartController"
+
+  def plist
+    <<~EOS
+      <?xml version="1.0" encoding="UTF-8"?>
+      <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+      <plist version="1.0">
+        <dict>
+          <key>EnvironmentVariables</key>
+          <dict>
+             <key>JAVA_OPTS</key>
+             <string>-Xms4G -Xmx8G -XX:+UseG1GC -XX:MaxGCPauseMillis=200 -Xloggc:gc-pinot-controller.log -Dlog4j2.configurationFile=/usr/local/etc/pinot/log4j2.xml</string>
+          </dict>
+          <key>KeepAlive</key>
+          <dict>
+            <key>SuccessfulExit</key>
+            <false/>
+          </dict>
+          <key>Label</key>
+          <string>#{plist_name}</string>
+          <key>ProgramArguments</key>
+          <array>
+            <string>/usr/local/bin/pinot-admin</string>
+            <string>StartController</string>
+            <string>-dataDir</string>
+            <string>#{var}/lib/pinot/data/controller</string>
+          </array>
+          <key>RunAtLoad</key>
+          <true/>
+          <key>WorkingDirectory</key>
+          <string>#{var}</string>
+        </dict>
+      </plist>
+    EOS
+  end
+
+  test do
+    assert_match(/StartController/, shell_output("#{bin}/pinot-admin -h 2>&1"))
```

Comment:
```suggestion
    assert_match("StartController", shell_output("#{bin}/pinot-admin -h 2>&1"))
```
Same comment re the test. Though, you never install anything into `bin`, so I'm surprised this even does anything.